src/App.tsx

```diff
@@ -1,35 +1,56 @@
 import React from 'react';
 import './App.scss';
-import { MoviesList } from './components/MoviesList';
+
 import moviesFromServer from './api/movies.json';
+import { MoviesList } from './components/MoviesList';
 
-type State = {};
+type State = {
+  inputValue: string,
+};
 
 export class App extends React.Component<{}, State> {
-  state: State = {};
+  state: State = {
+    inputValue: '',
+  };
+
+  changeInput = (event: React.ChangeEvent<HTMLInputElement>) => {
+    const { value } = event.target;
+
+    this.setState({ inputValue: value });
+  };
+
+  getFilteredMovies = (inputValue: string) => {
```

Comment:
you have inputValue in state, use it from there, no need to pass it into parameters.